### PR TITLE
Storage speedups

### PR DESCRIPTION
**Pull request by mrjackv**

### CODE REVIEW

File: share/revng/test/configuration/revng-c/segregate-stack-accesses.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ commands:
 
       revng analyze --resume "$OUTPUT" detect-stack-size "$INPUT" -o /dev/null;
 
-      revng artifact --resume "$OUTPUT" make-segment-ref "$INPUT" | FileCheck ${SOURCE}.filecheck.ll;
+      revng artifact --resume "$OUTPUT" make-segment-ref "$INPUT" |
+        zstdcat | llvm-dis | FileCheck ${SOURCE}.filecheck.ll;
```
Comment (aleclearmind) [inline share/revng/test/configuration/revng-c/segregate-stack-accesses.yml]: As a convention, we prefer `opt -S` to `llvm-dis`.

File: share/revng/test/configuration/revng-c/simplify-switch.yml [hunk 2]
```diff
@@ -15,5 +15,5 @@ commands:
       revng artifact
         --resume "$OUTPUT" --analyze simplify-switch
         --abi-enforcement-level=no "$INPUT" -o /dev/null ;
-      FileCheck
-        --input-file="$OUTPUT/simplify-switch/module.ll" "${SOURCE}.filecheck";
+      zstdcat "$OUTPUT/simplify-switch/module.ll" | llvm-dis |
```
Comment (aleclearmind) [inline share/revng/test/configuration/revng-c/simplify-switch.yml]: As a convention, we prefer `opt -S` to `llvm-dis`.

File: share/revng/test/configuration/revng-c/segregate-stack-accesses.yml (hunk 1 above)
Comment (mrjackv) [reply share/revng/test/configuration/revng-c/segregate-stack-accesses.yml]: Done

File: share/revng/test/configuration/revng-c/simplify-switch.yml (hunk 2 above)
Comment (mrjackv) [reply share/revng/test/configuration/revng-c/simplify-switch.yml]: Done

**Comment by aleclearmind** [comment]
Merged in 248e642. Thanks.

**Pull request closed without merging**